packages/local-refresh/src/index.ts

```diff
@@ -16,52 +16,58 @@ const progressBarFactory = () => {
   );
 };
 
-program.description("Script to refresh the local VOL application").action(async () => {
-  const actions = await Promise.all(
-    fs
-      .readdirSync(path.resolve(__dirname, "actions"))
-      .filter((file) => file.endsWith(".ts") && !file.endsWith("Interface.ts"))
-      .map((file) => import(`./actions/${file}`)),
-  );
+program
+  .description("Script to refresh the local VOL application")
+  .option(
+    "-i, --no-interaction",
+    "Run in non-interactive mode; this will execute the following: composer install, copy app dist files (all), reset database (full), create liquibase.properties file, flush redis, and reset ldap.",
```

Comment:
This will be quite likely to be overlooked during any additional action. I'd recommend we don't list out the current action and be generic.

E.g.
> Run in non-interactive mode; Does not ask any interactive question. All the actions will run with defaults.